docs/src/content/docs/api/tools.mdx

```diff
@@ -7,16 +7,57 @@ import { Badge, Tabs, TabItem } from '@astrojs/starlight/components';
 
 Tools in Ultravox allow you to extend the capabilities of your AI assistant by giving it access to external APIs and services. With tools, your AI can perform actions like fetching real-time data, interacting with databases, or integrating with third-party services, all while maintaining a natural conversation flow.
 
-### Two Types of Tools
-Ultravox supports durable and temporary tools. There is no functional difference between them. Durable tools are persisted and can be reused across calls or applications. Temporary tools are created each time you create a new Call and exist exclusively within the context of that call. (Temporary tools aren't visible in the ListTools response for example.)
+## Tools Quickstart
+The fastest way to get started with tools is to pass them in when you call [`Create Call`](../calls/#create-call) in the request body:
 
-Iteration is faster when using temporary tools. Durable tools shine once you have things dialed in, when you want to share tools across multiple applications, or if you have split responsibilities on the team.
+```javascript
+// Request body for creating an Ultravox call with a tool
+{
+  "model": "fixie-ai/ultravox-70B",
+  "systemPrompt": ...
+  "selectedTools": [
```

Comment:
This is fine as a quick start, but I think it'd be worth talking a bit about auth tokens and parameter overrides.  (Can also do that as a follow-up.)

Reply:
yes i think we will want to expand coverage there. will put on the list for follow-up.